### PR TITLE
proto: add tikv grpc interface for transaction debugger

### PR DESCRIPTION
**Pull request by choleraehyq**
@AndreMouche Does this interface definition good enough for our use?

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -226,3 +227,33 @@ message RawDeleteResponse {
     errorpb.Error region_error = 1;
     string error = 2;
 }
+
+message MvccPair {
```
Comment (zhangjinpeng1987) [inline proto/kvrpcpb.proto]: What does `MvccPair` means? A pair of what?

File: proto/kvrpcpb.proto [hunk 2]
```diff
@@ -226,3 +227,33 @@ message RawDeleteResponse {
...
+    bytes key = 2;
+}
+
+message KeyMvccResponse {
+    errorpb.Error region_error = 1;
+    string error = 2;
+    repeated MvccInfo infos = 3;
+}
+
+message StarttsMvccRequest {
```
Comment (siddontang) [inline proto/kvrpcpb.proto]: StartTs

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by choleraehyq** [comment]
@AndreMouche Updated.

**Comment by AndreMouche** [comment]
@disksing  @lishihai9017 @hhkbp2  PTAL

File: proto/kvrpcpb.proto [hunk 3]
```diff
@@ -226,3 +227,44 @@ message RawDeleteResponse {
...
+message MvccGetByStartTsRequest {
+    Context context = 1;
+    uint64 start_ts = 2;
+}
+
+message MvccGetByStartTsResponse {
+    errorpb.Error region_error = 1;
+    string error = 2;
+    bytes key = 3;
+    MvccInfo info = 4;
```
Comment (disksing) [inline proto/kvrpcpb.proto]: No `repeated`?

Comment (choleraehyq) [inline proto/kvrpcpb.proto]: MvccInfo include repeated WriteInfo and repeated ValueInfo.

Comment (disksing) [inline proto/kvrpcpb.proto]: I know. But should `GetByTs` returns mvcc values of multiple keys that belong to the same txn?

Comment (choleraehyq) [inline proto/kvrpcpb.proto]: No. Only the smallest (primary) key. See https://docs.google.com/document/d/11wo9UULumghZK0cey2_DJAKsd7-AgUwGEVDsUzoZwvQ/edit?ts=5958b3a8#

Comment (disksing) [inline proto/kvrpcpb.proto]: Okay. LGTM.

**Comment by hhkbp2** [comment]
LGTM

**Comment by lishihai9017** [comment]
LGTM

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**